src/main/java/jalasoft/com/servlets/TaskServlet.java

```diff
@@ -0,0 +1,18 @@
+package jalasoft.com.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class TaskServlet extends HttpServlet {
+	
+	protected void doPost(HttpServletRequest request, HttpServletResponse response)

```

Comment:
Multiple files have the same method doPost and doGet,maybe you can consider to add and interface

Reply:
I have a doubt here, since I thought this is like a convetional structure of any class that would extend from HttpServlet. For example when I create a Servlet file with the eclipse ide help, it automatically sets these methods and even asks if I want to add more methods like: doPut(), doDelete() and others like those. So this tells me that in case I need those kinds of methods I would just add them. But if I implement an interface I think I might be limiting the methods I would need in the future, even maybe going against "Implement Segregation" principle by forcing to take just two methods to this class. 
![image](https://user-images.githubusercontent.com/89421091/191643982-14d91af8-7a74-46f6-8cde-a65e712f8627.png)

Reply:
@Override annotacion added instead.